pkg/machinery/client/client.go

```diff
@@ -221,7 +233,63 @@ func (c *Client) GetConn(ctx context.Context, opts ...grpc.DialOption) (*grpc.Cl
 
 	dialOpts = append(dialOpts, opts...)
 
-	return grpc.DialContext(ctx, target, dialOpts...)
+	creds := BasicAuth{
+		auth: c.options.configContext.Auth.Basic.Username + ":" + c.options.configContext.Auth.Basic.Password,
```

Comment:
why here? unconditional?

Reply:
I'm trying to go step by step, compiling and trying. once I can successfully connect, authenticate and get a response I want to make it conditional etc.